lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -24,6 +24,11 @@ click_cancel_button:
   - selector:
       xpath: //button[contains(text(),'Cancel')]
     op: click
+click_button_type_on_page:
```

Comment:
How about adding a general action maybe named as`click_button` which can be used for any button clicking
`//button[@type='button' and contains(.,'<button_text>')]`

Reply:
I see we have many similar actions, only the button name is different. Adding a general action saves/reduces more duplicate lines.

Reply:
This rule is not very general actually, it only can be used for those type is button.  And I found that this kind of button were not as much as we thought. I can update the action name to `click_button`

Reply:
OK I will update